example-get-started-experiments/code/src/train.py

```diff
@@ -68,6 +68,7 @@ def train():
         models_dir = Path("models")
         models_dir.mkdir(exist_ok=True)
         learn.export(fname=(models_dir / "model.pkl").absolute())
```

Comment:
Is this still needed/useful? Seems odd to have `export` and `save` here.

Reply:
I would have to change the evaluate stage to use the `.pth` format. 
I was not able to make sagemaker work with the fast.ai exported in `.pkl` format.

Reply:
> Have you tried it? I would say let's do it if it's not that hard.

I assume it is possible but I didn't try to not include other changes